### PR TITLE
Added a bunch of sites

### PR DESCRIPTION
**Pull request by sprinkledcheese**

### CODE REVIEW

File: src/config/dark-sites.config [hunk 1]
```diff
@@ -78,6 +79,7 @@ checkra.in
 chessbomb.com
 cinema-city.pl
 clashofstats.com
+classic.minecraft.net
```
Comment (Gusted) [inline src/config/dark-sites.config:82]: White flash when loading......

File: src/config/dark-sites.config [hunk 2]
```diff
@@ -167,10 +172,12 @@ faceit.com
 factorio.com
 femto.dev
 femto.pw
+feudalife.indiegala.com
```
Comment (Gusted) [inline src/config/dark-sites.config:175]: Also a risky bright yellow.

File: src/config/dark-sites.config [hunk 3]
```diff
@@ -270,13 +279,15 @@ kiwifarms.net
 knockout.chat
 kodenames.io
 krunker.io
+lab.openbloc.fr/droneWorld/
```
Comment (Gusted) [inline src/config/dark-sites.config:282]: White.

File: src/config/dark-sites.config [hunk 4]
```diff
@@ -270,13 +279,15 @@ kiwifarms.net
...
 lainchan.org
 leakedsource.ru
 leakth.is
 letterboxd.com
 lightweightpdf.com
 linux.org.ru
 lookmovie.ag
+lordofthemanor.io
```
Comment (Gusted) [inline src/config/dark-sites.config:290]: Risky bright yellow.

File: src/config/dark-sites.config (hunk 2 above)
Comment (sprinkledcheese) [reply src/config/dark-sites.config:175]: Yes.

File: src/config/dark-sites.config (hunk 1 above)
Comment (sprinkledcheese) [reply src/config/dark-sites.config:82]: Will remove it, then.

File: src/config/dark-sites.config (hunk 3 above)
Comment (sprinkledcheese) [reply src/config/dark-sites.config:282]: Brave blocks something, has white flash, will remove it.

File: src/config/dark-sites.config (hunk 4 above)
Comment (sprinkledcheese) [reply src/config/dark-sites.config:290]: Dark reader makes the game dark.

Comment (Gusted) [reply src/config/dark-sites.config:287]: Ah Make sense,

**Pull request merged**